rules/new_tasks.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
this needs to be seriously abstracted. there has to be a module or class where a rule can be initialized based on the data in the rules model. rule execution needs to be celerified or at least developed with celery in mind